charts/api/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for the WBStack API
 name: api
-version: 0.31.2
+version: 0.32.0
```

Comment:
Maybe I missed something but why did you lower the version?

Reply:
it's a higher major version (31 vs 32)

Reply:
oh sorry misread